linux_os/guide/system/software/sudo/sudo_remove_no_authenticate/bash/shared.sh

```diff
@@ -0,0 +1,3 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_debian,multi_platform_ubuntu,multi_platform_ol
+
+sed -i '/^#/!s/\(.*\)![Aa][Uu][Tt][Hh][Ee][Nn][Tt][Ii][Cc][Aa][Tt][Ee]\(.*\)/\1\2/g' /etc/sudoers /etc/sudoers.d/*
```

Comment:
`sed -i '/^#/!s/\(.*\)!authenticate\(.*\)/\1\2/gI' /etc/sudoers /etc/sudoers.d/*` is simpler to understand

Reply:
I think that this check for upper case is not necessary. Sudo does not allow upper case !AUTHENTICATE, it will refuse to work.